### ISSUE TITLE
Error is not described to the user in text, if mandatory filed is left blank, Admin issue.

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#827]
**Describe the bug**
Error is not described to the user in text, if mandatory filed is left blank, Admin issue.

**To Reproduce**

Go to Admin  (http://localhost:9992/cm/app/?view=workflow&site=test&path=%2FSites%2Ftest&pathType=page)
Go to User Tab.
Click on Pencil icon "Edit user Details", for Admin User.
Blank out the Password and Confirm Password and then click Save.
No error is displayed on the page.
Same behavior for Add New User on the User Tab

**Expected behavior**
Error should be described to the user in text, if mendatory filed is left blank.

**Additional context**
Navigate all the options available for similar issue and fix.
